webapp/builder/views.py

```diff
@@ -0,0 +1,16 @@
+import flask
+
+
+def builder_blueprint():
```

Comment:
unless you think you think you'll pass parameters i don't think a function is really needed here.